docs/components/modeler/desktop-modeler/telemetry/telemetry.md

```diff
@@ -10,24 +10,7 @@ This page summarizes the data that is being collected.
 
 ## General structure of the events
 
-Independent from the type of the event we're dealing with, the payload we send to the ET has the following structure:
-
-```json
-{
-  "installation": "[THE_EDITOR_ID]",
-  "product": {
-    "name": "Camunda Modeler",
-    "version": "[MODELER_VERISON]",
-    "edition": "community",
-    "internals": {
-      "event": "[NAME_OF_THE_EVENT]",
-      "[SOME_ADDITIONAL_EVENT_DATA]": "[SOME_CUSTOM_VALUE]"
-    }
-  }
-}
-```
-
-Every event directly modifies the `internals` field of the payload.
+Most events sent share a similar payload which includes: the **event name**, the **application version** and the **editor ID**. ...
```

Comment:
The id would be something random, with a format like to " xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx". Since it's pretty random I'm not sure an example is necessary